enterprise/app/trends/trends_chart.tsx

```diff
@@ -51,9 +65,12 @@ const TrendsChartTooltip = ({
 export default class TrendsChartComponent extends React.Component {
   props: Props;
 
+  handleClick(date: any) {
+    router.navigateTo("/?start=" + date + "&end=" + date + this.props.hash);
```

Comment:
nit: if you replace `this.props.hash` with `(this.props.hash || "")` then you can avoid setting the empty `hash` prop on some of the charts